src/App.jsx

```diff
@@ -14,57 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const SelectedGoods = ({ selectedGood, onClear }) => {
+  if (!selectedGood) {
+    return (
+      <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+    );
+  }
 
+  return (
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
+      {selectedGood} is selected
+      <button
+        onClick={onClear}
+        data-cy="ClearButton"
+        type="button"
+        className="delete ml-3"
+      />
     </h1>
+  );
+};
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+const Good = ({ good, isSelected, onSelect }) => {
+  const buttonText = isSelected ? '-' : '+';
+  const buttonClass = isSelected ? 'is-info' : '';
+  const buttonCy = isSelected ? 'RemoveButton' : 'AddButton';
+  const goodBackground = isSelected ? 'has-background-success-light' : '';
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) for classes, not variables